playbooks/bind_config.yaml

```
@@ -2,6 +2,9 @@
 - name: Manage Bind9 configurations
   hosts: bind_servers
   become: true
+  gather_facts: false
+  vars:
+    bind_conf_files: '{{ bind_config__conf_files | d("files/bind/*") }}'
```

just for consistency: I have been using "private" variable names for pre-processed variables, to ensure they don't create conflicts

```suggestion
    _bind_conf_files: '{{ bind_config__conf_files | d("files/bind/*") }}'
```

---

Good idea. Will fix.